README.md

```diff
@@ -0,0 +1,8 @@
+nereid-review
+=============
+
+[![Build Status](https://travis-ci.org/openlabs/nereid-review.svg?branch=develop)](https://travis-ci.org/openlabs/nereid-review)
+[![Downloads](https://pypip.in/download/trytond_nereid_review/badge.svg)](https://pypi.python.org/pypi/trytond_trytond_nereid_review/)
+[![Latest Version](https://pypip.in/version/trytond_nereid_review/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_review/)
+[![Development Status](https://pypip.in/status/trytond_nereid_review/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_review/)
+
```

Comment:
There is no paragraph after this blank line, so there is no need for it.

Please remove it.